beacon_chain/genesis/src/beacon_state.rs

```diff
@@ -11,26 +10,13 @@ pub enum Error {
 }
 
 pub fn genesis_beacon_state(spec: &ChainSpec) -> Result<BeaconState, Error> {
-    /*
-     * Parse the ValidatorRegistrations into ValidatorRecords and induct them.
-     *
-     * Ignore any records which fail proof-of-possession or are invalid.
-     */
-    let validators = {
-        let mut inductor = ValidatorInductor::new(0, spec.shard_count, vec![]);
-        for registration in &spec.initial_validators {
-            let _ = inductor.induct(&registration, ValidatorStatus::Active);
-        }
-        inductor.to_vec()
-    };
-
     /*
      * Assign the validators to shards, using all zeros as the seed.
      *
      * Crystallizedstate stores two cycles, so we simply repeat the same assignment twice.
```

Comment:
we should remove this ref to `crystallized_state` -- i can make another PR if thats easier, i think the PR that did this clean up was already merged

Reply:
I removed this line, if that's what you were hoping for.